spec/services/authn_context_resolver_spec.rb

```diff
@@ -292,41 +292,181 @@
     end
 
     context 'IAL2 service provider' do
-      it 'uses the IAL ACR if one is present' do
-        service_provider = build(:service_provider, ial: 2)
-
-        acr_values = [
-          'http://idmanagement.gov/ns/assurance/aal/1',
-          'http://idmanagement.gov/ns/assurance/ial/1',
-        ].join(' ')
-
-        result = AuthnContextResolver.new(
+      let(:service_provider) { build(:service_provider, ial: 2) }
+      subject do
+        AuthnContextResolver.new(
           user: user,
           service_provider: service_provider,
           vtr: nil,
           acr_values: acr_values,
-        ).resolve
+        )
+      end
+      let(:result) { subject.resolve }
+
+      context 'IAL ACR value is present' do
```

Comment:
I find it helpful to have it read 'when an IAL ACR value is present'. From https://www.betterspecs.org/#contexts: 
> When describing a context, start its description with 'when', 'with' or 'without'.